CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 5.10.0 (2021-07-14)
+
```

Comment:
Can we bring back the subheadings for bug fixes and enhancements as this is a minor release?